src/main.cpp

```diff
@@ -1,8 +1,6 @@
 
 #include "stdafx.h"
 
-static_assert(wxCHECK_VERSION(3, 3, 0), "wxWidgets < 3.3.0 is not supported.");

```

Comment:
could make this 3.2?

Reply:
I pinned to the latest released version. Is this ok for you?